packages/perseus/src/interactive2/movable-line-options.js

```diff
@@ -3,8 +3,7 @@
 /**
  * A library of options to pass to add/draw/remove/constraints
  */
-import {vector as kvector} from "kmath";
-import {point as kpoint} from "kmath";
+import {vector as kvector, point as kpoint} from "kmath";
```

Comment:
Thanks for collapsing these!  Do you have tooling to help or just "brute force" 😁